locales-pending/premium.ftl

```diff
@@ -337,6 +337,16 @@ high-risk-breach-social-security-description = Scammers can open up new loans or
 high-risk-breach-social-security-step-one = Protect yourself by <link_to_info>setting up a fraud alert or freezing your credit.</link_to_info>
 high-risk-breach-social-security-step-two = <link_to_info>Check your credit report</link_to_info> for unrecognized accounts.
 
+# Social Security Number Modal
+
+ssn-modal-title = About fraud alerts and credit freezes
+ssn-modal-description-fraud-part-one = <b>A fraud alert</b> requires businesses to verify your identity before it issues new credit in your name. It’s free, lasts one year, and won’t negatively affect your credit score. 
+ssn-modal-description-fraud-part-two = To set one up, contact any one of the three credit bureaus. You don’t have to contact all three.
+ssn-modal-description-freeze-credit-part-one = <b>Freezing your credit</b> prevents anyone from opening a new account in your name. It’s free and won’t negatively affect your credit score, but you’ll need to unfreeze it before opening any new accounts. 
+ssn-modal-description-freeze-credit-part-two = To freeze your credit, contact each of the three credit bureaus — <link_to_equifax_info>Equifax</link_to_equifax_info>, <link_to_experian_info>Experian</link_to_experian_info>, and <link_to_transunion_info>TransUnion</link_to_transunion_info>.
```

Comment:
@flodolo Are these links OK, or do you think we need shorter tag names for l10n?
Ie: something like `<experian_link>Experian</experian_link>`

Reply:
Shorter would be better ;-)